src/core/link-to-dfn.js

```diff
@@ -76,6 +76,11 @@ export async function run(conf) {
 
   showLinkingError(badLinks);
 
+  // This needs to run before core/xref adds its data-cite and updates
+  // conf.normativeReferences and conf.informativeReferences.
+  await dataCite(conf);
```

Comment:
Failing to run this causes one test to fail in which an external definition (defined non-normatively) moves to normative references section.

Reply:
Need a better name for this function. Plus, move its definition to core/link-to-dfn